jekyll/_cci2/webhooks-reference.adoc

```diff
@@ -0,0 +1,632 @@
+---
+description: Reference page for webhooks payloads
+contentTags:
+  platform:
+  - Cloud
+  - Server v4.x
+  - Server v3.x
+---
+= Webhooks reference
+:page-layout: classic-docs
+:page-liquid:
+:icons: font
+:toc: macro
+:toc-title:
+
+[#common-top-level-keys]
+== Common top level keys of webhooks
+
+Each webhook will have some common data as part of the event:
+
+[.table.table-striped]
+[cols=3*, options="header", stripes=even]
+|===
+| Field
+| Description
+| Type
+
+| id
+| ID used to uniquely identify each event from the system (the client can use this to dedupe events)
+| String
+
+| happened_at
+| ISO 8601 timestamp representing when the event happened
+| String
+
+| webhook
+| A map of metadata representing the webhook that was triggered
+| Map
+|===
+
+NOTE: The event payloads are open maps, meaning new fields may be added to maps in the webhook payload without considering it a breaking change.
+
+[#common-sub-entities]
+== Common sub-entities of webhooks
+
+The following sections describe the payloads of different events offered with CircleCI webhooks. The schema of these webhook events will often share data with other webhooks. We refer to these common maps of data as "sub-entities". For example, when you receive an event payload for the `job-completed` webhook, it will contain maps of data for your *project, organization, job, workflow and pipeline*.
```

Comment:
```suggestion
The following sections describe the payloads of different events offered with CircleCI webhooks. The schema of these webhook events will often share data with other webhooks. These common maps of data are referred to as "sub-entities". For example, when you receive an event payload for the `job-completed` webhook, it will contain maps of data for your *project, organization, job, workflow and pipeline*.
```